test/com/facebook/buck/features/go/GoBinaryIntegrationTest.java

```diff
@@ -259,4 +259,12 @@ public void buildConstraints() throws IOException, InterruptedException {
     workspace.setUp();
     workspace.runBuckBuild("//:family").assertSuccess();
   }
+
+  @Test
```

Comment:
@mkaczanowski Thanks for landing this PR. Looks like this line disappeared during merging, effectively disabling this test. Could you fix it?

Reply:
ah, crap, thanks for spotting this! I'll fix this soon